src/mugs.js

```diff
@@ -293,7 +293,45 @@ define([
             rawControlXML: {
                 presence: 'optional',
                 lstring: 'Raw XML'
-            }
+            },
+            dataParent: {
+                lstring: 'Data Parent',
+                visibility: function(mug) {
+                    function recFunc(mug) {
+                        if (!mug) {
+                            return true;
+                        } else if (mug.__className === 'Repeat') {
+                            return false;
+                        }
+                        return recFunc(mug.form.controlTree.getNodeFromMug(mug).parent.value);
```

Comment:
What happens if this is called on a data-only mug (one that is in the data tree but not the control tree)? Will `getNodeFromMug(mug) -> null/undefined` and then `.parent -> Error`?